index.html

```diff
@@ -5,20 +5,38 @@
   <meta charset="UTF-8">
   <meta http-equiv="X-UA-Compatible" content="IE=edge">
   <meta name="viewport" content="width=device-width, initial-scale=1.0">
-  <link rel="stylesheet" href="style.css">
+  <link rel="stylesheet" href="./style.css">
   <link href='https://fonts.googleapis.com/css?family=Poppins' rel='stylesheet'>
   <title>Lucas' Portfolio</title>
 </head>
 
 <body>
+  <div class="mobile-menu" id="mobile-menu">
+    <div class="mobile-menu-header">
+      <button type="button" data-close-button class="close-button">&times;</button>
+    </div>
+    <nav>
+      <a href="#begin" data-close-button class="navigate">
+        Portfolio
+      </a>
+      <a href="#about-myself" data-close-button class="navigate">
+        About
+      </a>
+      <a href="#contact_form" data-close-button class="navigate">
+        Contact
+      </a>
+    </nav>
+  </div>
   <header>
     <div class="header-nav">
       <a href="#" class="my-logo">My Logo</a>
```

Comment:
- [ ] Please make sure to maintain the same margin and position of elements as shown in the design. Currently, the logo is flushed against the left edge of the screen unlike the design, which has margins around it as shown below:

- Your page
![image](https://user-images.githubusercontent.com/61377322/140861383-b6b842d0-e359-4409-9832-cf740cf0bceb.png)

- Figma design
![image](https://user-images.githubusercontent.com/61377322/140861397-63df5572-29b2-4deb-977d-1a6152d232df.png)